tutorials/haskell/hspec-golden-test-tutorial/tutorial.md

```diff
@@ -0,0 +1,506 @@
+---
+title: Golden Testing with Hspec-golden
+published: 2019-10-31
+ghc: 8.0.2
+lts: 14.10
+tags: haskell, golden, tests
+libraries: hspec-golden-0.1.0.1
+language: haskell
+author-name: Jorge Andrés Guerra Landázuri
+twitter-profile: Jorge_257
+github-profile: Jagl257
+description: Software testing is necessary to check if an application behaves in an expected way. Sometimes it is useful to store the expected output in a separate file. This is how golden tests work and in this tutorial we will be using the Hspec-golden haskell library to easily create and update our golden tests.
+---
+
+
+The value of software testing relies on the ability to check if an application behaves properly. Unit tests are the first layer of testing and help developers detect bugs early in the code by testing each component individually.
```

Comment:
@Jagl257 let's break long lines into multiple lines for easy comment/feedback. I think you use VIM so you can select the line and use `gq` to split them automatically 🚀